adaptation_layer/robotframework/environment/variables.txt

```diff
@@ -1,6 +1,6 @@
 *** Variables ***
 ${MSO-LO_HOST}      localhost    # Hostname of the MSO-LO
-${MSO-LO_PORT}      80	    # Listening port of the MSO-LO
+${MSO-LO_PORT}      8100	    # Listening port of the MSO-LO
```

Comment:
we prefer to keep 80 as MSO-LO_port value also during the tests

Reply:
The problem is EVER uses as component the Cloudify Manager app and this app uses the port 80 and 8080. So I need to map the MSO-LO port in a different value